python/tvm/relay/backend/contrib/uma/backend.py

```diff
@@ -278,7 +278,7 @@ def register(self) -> None:
         """
         registration_func = tvm.get_global_func("relay.backend.contrib.uma.RegisterTarget")
 
-        for name, attr in self._target_attrs:
+        for name, attr in self._target_attrs.items():
```

Comment:
Thanks for the fix @MichaelJKlaiber, just curious why this didn't get caught by https://github.com/apache/tvm/blob/main/tests/python/contrib/test_uma/test_target.py#L68?

Reply:
Maybe because this was too generic (though I expect that to be some sort of TypeError) : https://github.com/apache/tvm/blob/2aa0d1fbfcf4a31e343cc6852fdc4abd660c850a/tests/python/contrib/test_uma/test_target.py#L72.

Should we make sure that we catch the correct exception ?

Reply:
+1 to @manupak's suggestion; can we either use TVMError or a subclass of that?